src/styles/main.scss

```diff
@@ -1,3 +1,57 @@
+@import "./utils/variables";
+
 body {
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+
+  width: (($daySize + $gapSize) * 7);
+  margin: 25% auto;
```

Comment:
not the greatest positioning technique
if you nest your calendar inside flex container or at least style the body properly - it would be properly centered even without margins applied for `.calendar`
```suggestion
```

```
body {
    margin: 0;
    display: flex;
    height: 100vh;
    justify-content: center;
    align-items: center;
}
```